config/stack.go

```diff
@@ -96,17 +342,17 @@ func ReadStackConfigFile(ctx context.Context, opts *options.TerragruntOptions) (
 	return config, nil
 }
 
-// WriteUnitValues generates and writes unit values to a terragrunt.values.hcl file in the specified unit directory.
+// writeUnitValues generates and writes unit values to a terragrunt.values.hcl file in the specified unit directory.
 // If the unit has no values (Values is nil), the function logs a debug message and returns.
```

Comment:
NIT: I don't think we should be logging debug messages like this for common-case. It's going to make debugging annoying.